cmd/registry/cmd/upload/bulk/openapi.go

```diff
@@ -174,8 +174,7 @@ func (task *uploadOpenAPITask) populateFields() error {
 	versionPart := parts[len(parts)-2]
 	task.versionID = sanitize(versionPart)
 
-	specPart := parts[len(parts)-1]
-	task.specID = sanitize(specPart)
+	task.specID = "openapi"
```

Comment:
Why make this a field on the struct at all anymore?

Reply:
good question - I guess I was subconsciously uncommitted to the permanence of this :) 

I'll remove it before merging